sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/ColumnVector.java

```diff
@@ -801,6 +801,14 @@ public final int appendFloats(int count, float v) {
     return result;
   }
 
+  public final int appendFloats(int length, float[] src, int offset) {
```

Comment:
Yep. That's the reason why this is missing until now. We need to use them in order to `ColumnarBatch` independently from Parquet.